EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -999,12 +999,12 @@ QUEST_LV_0300_20160310_000998	It's for the wellbeing of the people here so I beg
 QUEST_LV_0300_20160310_000999	I thank you.{nl}My first request is for you to gather the blood of Green Eldigos.{nl}
 QUEST_LV_0300_20160310_001000	I need to extract the blood from a live Green Eldigo for my research, but that's no easy task.{nl}I'll give you some Tranquilizer Powder to knock {nl}the Green Eldigos out and bring some blood back.
 QUEST_LV_0300_20160310_001001	So, how did it go?{nl}
-QUEST_LV_0300_20160310_001002	Very good. Excellent.{nl}I think I'd be confident in asking you to do more tasks.{nl}
-QUEST_LV_0300_20160310_001003	I'll go to the Forest of Prayer Entrance and prepare for the final experiment.{nl}You go and gather the ingredients needed for the last experiment.{nl}
+QUEST_LV_0300_20160310_001002	Very good. Excellent.{nl}With the skills you've shown me, I think I can be confident in assigning you more tasks.{nl}
+QUEST_LV_0300_20160310_001003	I'll go to the Forest of Prayer's Entrance and prepare for the final experiment.{nl}You can go and gather the ingredients needed for the last experiment.{nl}
```

Comment:
Forest of Prayer's Entrance -> Forest of Prayer Entrance

It's a map name, by adding an apostrophe you are deviating from the map name.